index.html

```diff
@@ -4,17 +4,17 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <title>Tasnim's Portfolio</title>
```

Comment:
- Good job done so far, according to the project requirements, `if issues were found,  please link to the corresponding line of code in the Pull Request changes`. Please implement these for all changes made 👍 👍